### PR TITLE
Create SitePackage.lua tried on Bright sites

### PR DESCRIPTION
**Pull request by fgeorgatos**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
this deprecates #269 & #268

File: contrib/Bright/SitePackage.lua [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+   --  c) Write the same information directly to some database.
+
+   -- This is an example writing to syslogd:
+
+   if (mode() ~= "load") then return end
+   local user        = os.getenv("USER")
+   local host        = uname("%n")
+   local currentTime = epoch()
+-- 20170503FG improve syslog 4 SGE jobs, but make it flexible, if variable is not there
+   local jobid       = os.getenv("JOB_ID")
```
Comment (wpoely86) [inline contrib/Bright/SitePackage.lua]: ` or "unknown"` is a bit more elegant

File: contrib/Bright/SitePackage.lua [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+function report_loads()
+
+   local sys         = os.getenv("LMOD_sys") or "Linux"
+   if (sys == "Linux") then
+      local a = s_msgA
+      for i = 1,#a do
+         local msg = a[i]
+         os.execute("logger -t ModuleUsageTracking -p local0.info " .. msg)
```
Comment (wpoely86) [inline contrib/Bright/SitePackage.lua]: you want add ` -- ` before appending message or you might get trouble with unloads (`-GCC`)

Comment (rtmclay) [inline contrib/Bright/SitePackage.lua]: But this is a load hook so it won't be called when unloading.

**Comment by rtmclay** [comment]
This is merged and I made the change that @wpoely86 suggested.  All of this has been updated to HEAD.

File: contrib/Bright/SitePackage.lua (hunk 2 above)
Comment (fgeorgatos) [inline contrib/Bright/SitePackage.lua]: @wpoely86 : thanks, that was a good catch!

**Pull request merged**